vizro-core/examples/visual-vocabulary/pages/examples/sparkline.py

```diff
@@ -0,0 +1,20 @@
+import vizro.models as vm
+import vizro.plotly.express as px
+from vizro import Vizro
+
+stocks = px.data.stocks()
+
+def sparkline(df, x, y):
+    fig=px.area(df, x, y, facet_row="variable", height=100)
+    fig.update_xaxes(showticklabels=False, showgrid=False, zeroline=False)
+    fig.update_yaxes(showticklabels=False, showgrid=False, zeroline=False)
+    fig.update_layout(margin=dict(l=0, r=0, t=0, b=0), showlegend=False)
+    return fig
+
+page = vm.Page(
+    title="Sparkline",
+    components=[vm.Graph(figure=sparkline(stocks, x="date", y=["GOOG", "AMZN"]))],
+)
+
+dashboard = vm.Dashboard(pages=[page])
```

Comment:
You were already very close in solving this! A couple of things I've changed:

- Added the`@capture(graph)` decorator above your function such that all post-updates have an effect
- Simplified the function call by getting the data set in the right format

```suggestion
import vizro.models as vm
import vizro.plotly.express as px
from vizro import Vizro
from vizro.models.types import capture

stocks = px.data.stocks()
stocks_melt = stocks.melt(id_vars='date', value_vars=['GOOG', 'AMZN', "AAPL"], var_name='stock', value_name='value')


@capture("graph")
def sparkline(data_frame, x, y, facet_row):
    fig=px.line(data_frame, x=x, y=y, color=facet_row, facet_row=facet_row)
    fig.update_xaxes(visible=False)
    fig.update_yaxes(visible=False)
    fig.update_layout(showlegend=False)
    return fig

page = vm.Page(
    title="Sparkline",
    components=[vm.Graph(figure=sparkline(stocks_melt, x="date", y='value', facet_row="stock"))],
)

dashboard = vm.Dashboard(pages=[page])
```

Reply:
The stocks_melt data set you would have to add to the `_pages_utils.py` :)